src/systems/ec/implementations/react/packages/site-header/SiteHeader.jsx

```diff
@@ -70,7 +87,6 @@ SiteHeader.defaultProps = {
   logo: {
     title: '',
     alt: '',
-    language: '',
```

Comment:
We should already add the possibility to have different languages (as you did for EU).
For now we only have the EN version of the logo, but it will be translated for sure.

Reply:
We'd have a new variable, not used anywhere, eslint complaining about that... Are you sure you really want this? 

If the designers provide translated logos, then we could update the component. It won't be a BC since we would just add a new property.